packages/e2e-tests/src/specs/editor/shopping/productMenu.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import {
+  createNewStory,
+  withExperimentalFeatures,
+  minWPVersionRequired,
+  withPlugin,
+} from '@web-stories-wp/e2e-test-utils';
+
+describe('Shopping product', () => {
```

Comment:
Started a test --- this depends on https://github.com/GoogleForCreators/web-stories-wp/pull/11250 to get the products setup